saleor/csv/utils/export.py

```diff
@@ -0,0 +1,345 @@
+import os
+from collections import ChainMap, defaultdict
+from tempfile import NamedTemporaryFile
+from typing import IO, TYPE_CHECKING, Dict, List, Optional, Set, Tuple, Union
+
+import petl as etl
+from django.conf import settings
+from django.db.models import F
+from django.utils import timezone
+
+from ...celeryconf import app
+from ...core.utils import build_absolute_uri
+from ...product.models import Product, ProductVariant
+from .. import JobStatus
+from ..emails import send_email_with_link_to_download_csv
+from ..models import Job
+
+if TYPE_CHECKING:
+    # flake8: noqa
+    from django.db.models import QuerySet
+
+
+class ProductExportFields:
+    """Data structure with fields for product export."""
+
+    PRODUCT_RELATION_FIELDS = [
+        "attribute_value",
+        "attribute_slug",
+        "collections__slug",
+        "image_path",
+    ]
+    WAREHOUSE_FIELDS = [
+        "stock__warehouse__slug",
+        "stock__quantity",
+        "stock__quantity_allocated",
+    ]
+    ATTRIBUTE_FIELDS = ["slugs_of_values", "attribute_slug"]
+    PRODUCT_HEADERS_MAPPING = {
+        "product": {
+            "id": "id",
+            "name": "name",
+            "description": "description",
+            "product_type__name": "product type",
+            "category__slug": "category",
+            "is_published": "visible",
+            "price_amount": "price",
+            "product_currency": "product currency",
+        },
+        "product_many_to_many": {"collections": "collections"},
+        "variant": {
+            "sku": "sku",
+            "price_override_amount": "price override",
+            "cost_price_amount": "cost price",
+            "variant_currency": "variant_currency",
+        },
+        "common": {"images": "images"},
+    }
+
+
+MEDIA_URL = settings.MEDIA_URL
```

Comment:
Are you sure you need it here? I don't like it too much as it forces the lazy object to be resolved during import, which could potentially lead to unwanted side-effects

Reply:
No, I don't need it here. I'll remove this global variable.